.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+on: [pull_request, workflow_dispatch]
+jobs:
+  # Xcode 12 Jobs
+  unit_test_job:
+    name: Unit (Xcode 12.5)
+    runs-on: macOS-11
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Use Xcode 12.5
+        run: sudo xcode-select -switch /Applications/Xcode_12.5.app
+      - name: Install CocoaPod dependencies
+        run: pod install
+      - name: Run Unit Tests
+        run: xcodebuild -workspace 'PopupBridge.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UnitTests' -destination 'name=iPhone 12,platform=iOS Simulator' test #TODO: add some sort of xcpretty or xcbeautify
+  ui_test_job:
+    name: UI (Xcode 12.5)
+    runs-on: macOS-11
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Use Xcode 12.5
+        run: sudo xcode-select -switch /Applications/Xcode_12.5.app
+      - name: Install CocoaPod dependencies
+        run: pod install
+      - name: Run UI Tests
+        run: xcodebuild -workspace 'PopupBridge.xcworkspace' -sdk 'iphonesimulator' -configuration 'Release' -scheme 'UITests' -destination 'name=iPhone 12,platform=iOS Simulator' test #TODO: add some sort of xcpretty or xcbeautify
+
+  # Xcode 13-beta Jobs
+  ui_test_job_xcode_beta:
```

Comment:
I only added the UI tests for XCode 13, in addition to 12.5, since I don't think it's that valuable to unit test both XCode versions.